core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -459,7 +459,7 @@ private[spark] class Executor(
         case CausedBy(cDE: CommitDeniedException) =>
           val reason = cDE.toTaskFailedReason
```

Comment:
we should probably change this to be toTaskCommitDeniedReason since its not failed anymore.